jaeger-core/src/main/java/com/uber/jaeger/samplers/RemoteControlledSampler.java

```diff
@@ -148,6 +148,17 @@ public boolean equals(Object sampler) {
   }
 
   @Override
+  public int hashCode() {
```

Comment:
presumably things like this are because of presence of equals methods. Are we expecting people to try and do equivalence on types like this? If not, an easier path is to remove equals